contracts/test/v0.8/dev/ArbitrumSequencerUptimeFeed.test.ts

```diff
@@ -306,7 +306,7 @@ describe('ArbitrumSequencerUptimeFeed', () => {
       // Assert update
       expect(await arbitrumSequencerUptimeFeed.latestAnswer()).to.equal(1)
       expect(updateTx.cumulativeGasUsed.toNumber()).to.be.closeTo(
-        93137,
+        93015,
```

Comment:
does this belong in this PR?

Reply:
Yes, Interestingly. Hardhat uses the highest supported Solidity version within a contract's defined pragma to run its tests against. 

It turns out this contract has a wide pragma, so these tests used to run on a 0.8.6 compiled contract, and now get run on 0.8.13, which has some gas optimizations.

Reply:
:+1: